pkg/crc/machine/machine.go

```diff
@@ -153,11 +193,17 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		result.Status = vmState.String()
 	}
 
+	err = fillClusterConfig(crcBundleMetadata, &result.ClusterConfig)
+	if err != nil {
+		result.Error = err.Error()
+		return *result, err
```

Comment:
Same here `err` should be wrapped.

Reply:
The only error which fillClusterConfig returns is already created with errors.Newf(). Or should we forbid the use of errors.Newf() in subfunctions, and only allow it in Start() right after result.Error was set?

Reply:
@cfergeau yes, we forbid to use of `errors.Newf()` for the subfunctions and only use it to functions which are directly interacted with cmd side (i.e. start, stop ..etc.)